documentation/modules/exploit/windows/fileformat/office_excel_slk.md

```diff
@@ -0,0 +1,49 @@
+## Introduction
+
+The .slk file format used by Microsoft Excel has the ability to execute local commands via the `EEXEC(cmd)` function. 
+This module takes advantage of this 'feature' to run a download-and-execute powershell command in order to spawn a session
+on the target.
+
+## Vulnerable Application
+
+  Microsoft Excel
+
+## Verification Steps
+
+  Example steps in this format (is also in the PR):
+
+1. Start `msfconsole`
+2. `use exploit/windows/fileformat/office_excel_slk`
+3. `set LHOST 192.168.x.x`
+4. `run`
+5. Execute generated file and press 'Enable Content' in Excel
+
+## Options
+
+  **FILENAME**
+
+  The name of the generated .slk file.
```

Comment:
if there is a default, add that here, something like  `.slk file.  Default is a randomly generated file name`